### PR TITLE
Fix #4105: Fixed Enter Pin Text View in MDPI small

### LINKED ISSUE #4105: Pin Text View is not visible in MDPI small
Pin Text View is not visible 
Screenshot_1642243720
MDPI_SMALL
Size: 3.2 inch
Android API: 28

### PR DESCRIPTION
**Pull request by deepanshu731**
Fixed #4105 

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

-------------------------------------------------------------------------------------------------------------------------------------------------------------------------
| Before | After |
|----------|---------|
|Screenshot_20220214_225209|Screenshot_20220214_221242|
|Screenshot_20220214_225230|Screenshot_20220214_221229|
|Screenshot_20220214_225034|Screenshot_20220214_221355|
|Screenshot_20220214_225021|Screenshot_20220214_221345|
|Screenshot_20220214_225426|Screenshot_20220214_221044|
|Screenshot_20220214_225416|Screenshot_20220214_221108|

### CODE REVIEW

File: app/src/main/res/layout/pin_password_activity.xml [hunk 1]
```diff
@@ -56,146 +56,153 @@
...
-      </androidx.constraintlayout.widget.ConstraintLayout>
       </ScrollView>
       <View
         android:layout_width="match_parent"
         android:layout_height="6dp"
         android:background="@drawable/toolbar_drop_shadow" />
     </FrameLayout>
   </androidx.constraintlayout.widget.ConstraintLayout>
-</layout>
+</layout>
```
Comment (rt4914) [inline app/src/main/res/layout/pin_password_activity.xml]: EOF is missing, add one blank line at the end.

**Review by rt4914 — CHANGES_REQUESTED** [review]
Please add before/after screenshot in all qualifiers like 
mobile-portrait
mobile-landscape
tablet-portrait
tablet-landscape

So that we can be sure that there is no change in UI for other qualifiers.

**Comment by rt4914** [comment]
@deepanshu731 Assign this PR to me once its ready for review again.

File: app/src/main/res/layout/pin_password_activity.xml (hunk 1 above)
Comment (deepanshu731) [reply app/src/main/res/layout/pin_password_activity.xml]: done

**Comment by deepanshu731** [comment]
@rt4914 i have added enter pin edit text view and show pin image in a horizontal linear layout and added weight to make the UI and given constraints accordingly .
There are few changes in UI in landscape of mdpi (3.2 inch size) and  pixel 3a (5.5 inches size) , i think afterward UI is better as its more linearly aligned , so what are your thoughts on this ?

**Comment by rt4914** [comment]
@deepanshu731 Unassigning myself for now. As discussed you can coordinate with @rishidyno , @vrajdesai78 and @adizcode about how to work on such UIs which can be creating in one single xml file with the usage of `-1px` and `-2px` values.

**Pull request closed without merging**